extensions/replication/ReplicationQueuePopulator.js

```diff
@@ -23,9 +23,23 @@ class ReplicationQueuePopulator extends QueuePopulatorExtension {
         if (queueEntry.getReplicationStatus() !== 'PENDING') {
             return;
         }
+
         this.publish(this.repConfig.topic,
                      `${queueEntry.getBucket()}/${queueEntry.getObjectKey()}`,
                      JSON.stringify(entry));
+
+        if (queueEntry.getReducedLocations) {
```

Comment:
For info:

Although for now only entries to be replicated go through here, and they all should be of type `QueueEntry` which has `getReducedLocations` function, the pending PR for echo mode splits this class into ObjectQueueEntry and BucketQueueEntry, depending on whether the log entry is an object update or bucket creation.

In such case, instead of doing this metrics computation here, it can make more sense to have a `ObjectQueueEntry.getMetrics()` and possibly `BucketQueueEntry.getMetrics()` methods (or `updateMetrics()` methods if it's more appropriate), because at the end the metrics you want will depend on the type of log entry processed. So the logic of checking the presence of `getReducedLocations` may have to be changed then.

For now, let's leave as-is though.

Reply:
Just to note, I've added a method `getBytesMetric` to `ObjectQueueEntry`. For now, I am not collecting `BucketQueueEntry` as I'm unsure on how we want to display this metric. Do we just collect the number of bucket entries and display replicated buckets? Also, I believe this only applies to Echo